CHANGELOG.rst

```diff
@@ -1,7 +1,7 @@
 CHANGELOG
 =========
 
-0.5.0 (unreleased)
+0.5.0 (2018-02-07)
```

Comment:
Shall we go to 1.0.0 now that it is used in production for a while?

Reply:
Yes, OK!